### ISSUE TITLE
Linux based NCM device shows up "UsbNcm Host Device", ignoring iManufacturer and iProduct

### ISSUE THREAD

**Issue by aloncatz** [microsoft/NCM-Driver-for-Windows#21]
I have a linux based NCM device. On Mac and Linux it shows up with the correct manufacturer and name. On Windows it shows up as "UsbNcm Host Device". Am I doing something wrong or is it a limitation of the driver?

**Comment by mengxp** [comment]
because the driver inf file is microsoft default one, not you custom

**Comment by aloncatz** [comment]
Yes, but I think the default driver/inf file should be using iManufacturer
and iProduct instead of just saying "UsbNcm Host Device". It looks like it
was the intention according to the code .
